src/main_app/components/NavBar/MainMenu/index.js

```diff
@@ -45,19 +45,19 @@ function MainMenu ({ isOpen, className }) {
             <a href='#vai'>Validate an idea</a>
           </li>
           <li>
-            <SVGIcon name='offerings/design' />
+            <SVGIcon name='offerings/design' className={classes.validateIdeaLogo}/>
```

Comment:
Can we change this class name to something more generic? This name is not appropriate to use it on all icons.